schemas/CQLSchema.schema

```diff
@@ -5,31 +5,37 @@
   "type": "object",
   "properties": {
     "name": {
-      "type": "string"
+      "type": "string",
+      "description": "Name of the property"
     },
     "description": {
-      "type": "string"
+      "type": "string",
+      "description": "Description of the property"
     },
     "prefix": {
-      "type": "string"
+      "type": "string",
+      "description": "Prefix of the property"
     },
     "uri": {
-      "type": "string"
+      "type": "string",
+      "uri": "Uri of the property"
```

Comment:
to fix jenkins `ERROR: lint-raml-cop: CQLSchema.schema: Missing "description" for: name, type, uri`
we probably should add 
`"description": "Uri of the property"`